src/setup.c

```diff
@@ -350,8 +352,8 @@ static void setup_timer1(void)
 	timer_set_mode(TIM1, TIM_CR1_CKD_CK_INT, TIM_CR1_CMS_EDGE,
 		       TIM_CR1_DIR_UP);
 	timer_set_clock_division(TIM1, 0x00);
-	timer_set_prescaler(TIM1, (rcc_apb2_frequency / 100));
-	timer_set_period(TIM1, 0x1);
+	timer_set_prescaler(TIM1, (rcc_apb2_frequency / 160000 - 1));
+	timer_set_period(TIM1, 0x9);
```

Comment:
I'd say `10 - 1` instead. I think it is more clear (just like you used `160000 - 1`).

Reply:
Changed